bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/commonChannels.xml

```diff
@@ -469,4 +469,18 @@
 			</options>
 		</state>
 	</channel-type>
+	<channel-type id="dreameControl">
+		<item-type>String</item-type>
+		<label>Control Vacuum</label>
+		<state>
```

Comment:
Does this channel also show the state of the control  or is it only to send commands. In the latter case this should be `<command>` instead of `<state>`.

Reply:
Ah, did not know that.
it is the latter (for now) as it is unclear still how the vacuum is precisely reporting back the current activity.